Project/app/src/main/java/org/ionproject/android/SharedViewModel.kt

```diff
@@ -0,0 +1,21 @@
+package org.ionproject.android
+
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModel
+import org.ionproject.android.common.model.ClassSummary
+import org.ionproject.android.common.model.CourseSummary
+
+class SharedViewModel : ViewModel() {
+
+    // Search text used to pass data from search bar to searchResultFragment
```

Comment:
BTW, this is not KDoc, but it should be. I also have doubts regarding the use of lateinit here. Shouldn't it be nullable instead?

Reply:
With the way the navigation has been setup within the app, I don't think it's possible to ever not have a value in one of those fields, but, making them nullable would improve legibility for sure, I hadn't thought about that.